pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -29,7 +28,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	policy "k8s.io/api/policy/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
+
```

Comment:
could you keep the one without an alias?

Reply:
Done